lib/vsc/install/shared_setup.py

```diff
@@ -24,19 +24,46 @@
 import shutil
 import sys
 import re
+
+try:
+    from unittest.suite import TestSuite
+except ImportError:
+    from unittest import TestSuite
+
 from distutils import log  # also for setuptools
 from distutils.dir_util import remove_tree
 
+from setuptools.command.test import test as TestCommand
+from setuptools.command.test import ScanningLoader
+
 # 0 : WARN (default), 1 : INFO, 2 : DEBUG
 log.set_verbosity(2)
 
 has_setuptools = None
 
+# available authors
+ag = ('Andy Georges', 'andy.georges@ugent.be')
+eh = ('Ewan Higgs', 'Ewan.Higgs@UGent.be')
+jt = ('Jens Timmermans', 'jens.timmermans@ugent.be')
+kh = ('Kenneth Hoste', 'kenneth.hoste@ugent.be')
+kw = ('Kenneth Waegeman', 'Kenneth.Waegeman@UGent.be')
+lm = ('Luis Fernando Munoz Meji?as', 'luis.munoz@ugent.be')
+sdw = ('Stijn De Weirdt', 'stijn.deweirdt@ugent.be')
+wdp = ('Wouter Depypere', 'wouter.depypere@ugent.be')
+wp = ('Ward Poelmans', 'Ward.Poelmans@UGent.be')
```

Comment:
some alphabet, different sorting key (variable name vs actual name of person), so nvm :)